k8s/node-mdb-ss.yaml

```diff
@@ -76,12 +81,6 @@ spec:
             port: bdb-port
           initialDelaySeconds: 15
           timeoutSeconds: 10
-#        readinessProbe:
-#          httpGet:
-#            path: /
-#            port: bdb-port
-#          initialDelaySeconds: 15
-#          timeoutSeconds: 10
```

Comment:
That's not my understanding of what a readinessProbe is for.

"The kubelet uses readiness probes to know when a Container is ready to start accepting traffic. A Pod is considered ready when all of its Containers are ready. One use of this signal is to control which Pods are used as backends for Services. When a Pod is not ready, it is removed from Service load balancers." from https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/ 

Suppose a container is live (i.e. it's running fine), and it's been handling traffic fine ("ready") for a long time. Then something goes wrong, the container is still running fine (i.e. it's livenessProbe is working), but it's no longer handling incoming traffic properly for some reason. It's readinessProbe would fail, and that would tell all Services to stop sending traffic to that pod.

I wonder if we should use an exec probe instead of an httpGet probe for livenessProbes, i.e. a more OS-level check that doesn't rely on some webserver working inside the container: that's what the readinessProbe should be checking. The above link has a nice example using `cat`.